index.js

```diff
@@ -7,8 +7,24 @@ module.exports = {
   included: function(app) {
     this._super.included(app);
 
-    if (app.env === 'test' || app.env === 'development') {
+    this.app = app;
+    this.addonConfig = this.app.project.config(app.env)['ember-cli-page-object'] || {};
+    this.addonBuildConfig = this.app.options['ember-cli-page-object'] || {};
```

Comment:
Do we need to read the configuration? We're not adding new options for the add-on.